syntax = "proto2";

package fueling.common.record.kinglong.proto.cybertron;

message MetricAlarm {
    optional string metric_name=1;
    optional uint64 mode_num=2;
    optional uint64 fault_num=3;
    optional string msg=4;
    optional uint64 stamp=5;    
}
